app/settings/profile/name/-components/citation-preview/component.ts

```diff
@@ -37,7 +34,7 @@ export default class CitationPreview extends Component {
         return returnValue;
     }
 
-    @computed('user.familyName', 'user.givenName', 'user.middleNames', 'user.suffix')
+    @computed('user.{familyName,givenName,middleNames,suffix}')
```

Comment:
that's cool, I like this.

Reply:
Me too! It's always been on, but previously it didn't understand that `@computed()` was defining a computed (it was not aware of the computed decorator).